Sources/Rendering/Core/Property/index.d.ts

```diff
@@ -223,6 +225,16 @@ export interface vtkProperty extends vtkObject {
    */
   getMetallicTexture(): vtkTexture;
 
+  /**
+   * Get the roughness & metallic texture.
+   */
+  getRMTexture(): vtkTexture;
+
+  /**
+   * Get the occlusion, roughness & metallic texture.
+   */
+  getORMTexture(): vtkTexture;
```

Comment:
It looks like these getters should return a Nullable vtkTexture type.

Reply:
done